docusaurus.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
   favicon: 'img/favicon.ico',
   organizationName: 'electron',
   projectName: 'electron',
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en', 'es-ES']
```

Comment:
Testing the Spanish one for now although all the translations are pulled and organized.
We could keep this in sync automatically with Crowdin somehow (during the prebuild step probably?)